lib/ReactViews/Workbench/Controls/legend.scss

```diff
@@ -5,14 +5,13 @@
   composes: list-reset from "../../../Sass/common/_base.scss";
   font-family: $font-mono;
   font-size: $font-size-small;
+  // Small font size prevents the font from dictating the table row height.
+  font-size: 5px;
```

Comment:
How do these two font-size properties interact?

Reply:
The second overrides the first/
![Screenshot 2024-08-15 at 3 46 27 pm](https://github.com/user-attachments/assets/1638df55-365d-4aac-beb3-e803563e1671)

The actual font-size of legends gets set by legend-title below, I can't find a way to get text with this rule applied. Given 5px is unreadable my vote would be to either a) remove both, b) keep `$font-size-small`